bibcheck/plugins/fix_double_longitude.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Amend institution records with no latitude and two longitudes.
+
+Some institution records have no latitude and two longitudes.
+This plugin fixes this data issue by deleting the longitudes
+and assigning a new latitude/longitude pair using the Google
+Maps geocoding API.
+
+We expect to run this plugin only once; moreover, we expect
+to call the Google Maps API at most a couple hundred times,
+well under the limit of a thousand requests per day.
+"""
+
+import os
+
+import googlemaps
+
+
+CFG_GOOGLE_MAPS_API_KEY = os.environ.get('CFG_GOOGLE_MAPS_API_KEY')
+
+
+def geocode(record):
+    """Uses the Google Maps API to geocode an institution.
+
+    Uses only information about the city the institution is in
+    because it's the only one which is reliably there. In fact,
+    the queries
+
+        034__d:** and not 034__f:**
+
+    and
+
+        371__b:** and 034__d:** and not 034__f:**
+
+    return the same set of results.
+    """
+    def _get_approximate_address(record):
+        return record.iterfield('371__b')[0]
+
+    gmaps = googlemaps.Client(key=CFG_GOOGLE_MAPS_API_KEY)
```

Comment:
Can you move this initialization at the global level. Anyway it's going to be executed in a single process.